draft-ietf-ppm-dap.md

```diff
@@ -1627,32 +1618,27 @@ In general, aggregation cannot begin until the Collector specifies a query and
 an aggregation parameter. However, depending on the VDAF and batch mode in use,
 it is often possible to begin aggregation as soon as reports arrive. For
 example, Prio3 has just one valid aggregation parameter (the empty string), and
-thus allows for eager aggregation; and both the time-interval
-({{time-interval-batch-mode}}) and leader-selected
-({{leader-selected-batch-mode}}) batch modes defined in this document allow for
-eager aggregation.
+thus allows for eager aggregation. Both the time-interval and leader-selected
+batch modes defined in this document ({{batch-modes}}) allow for eager
+aggregation.
 
-An aggregation job can be thought of as having three phases, which are
-described in the remaining subsections:
+An aggregation job has three phases:
 
 - Initialization: Begin the aggregation flow by disseminating report shares and
   initializing the VDAF prep state for each report.
 - Continuation: Continue the aggregation flow by exchanging prep shares and
   messages until preparation completes or an error occurs.
-- Completion: Finish the aggregation flow, yielding an output share
-  corresponding to each report share in the aggregation job, and update the
-  appropriate aggregate shares with these output shares.
+- Completion: Finish the aggregation flow, yielding an output share for each
+  report share in the aggregation job.
 
 After an aggregation job is completed, each Aggregator updates running-total
 aggregate shares and other values for each "batch bucket" associated with a
-recovered output share, as described in {{batch-buckets}}. These values are
+prepared output share, as described in {{batch-buckets}}. These values are
 stored until the batch is collected as described in {{collect-flow}}.
 
-Apart from VDAF preparation, another important task of the aggregation
-interaction is to provide replay protection ({{replay-protection}}). Along with
-the output shares, each Aggregator records the IDs of all reports it is has
-aggregated for a given task: before committing to an output share, it checks
-whether the corresponding report ID is in the set of stored IDs.
+The aggregation interaction provides replay protection ({{replay-protection}}).
+Before committing to an output share, Aggregators check whether the
+corresponding report ID has already been aggregated.
```

Comment:
Prescribe what, not how. What we care about is a report ID not being aggregated twice. We don't have to spell out anything about storage.